tensorflow/tools/docker/Dockerfile

```diff
@@ -29,8 +30,8 @@ RUN pip --no-cache-dir install \
 
 # Install TensorFlow CPU version.
 ENV TENSORFLOW_VERSION 0.7.1
-RUN pip --no-cache-dir install \
-    http://storage.googleapis.com/tensorflow/linux/cpu/tensorflow-${TENSORFLOW_VERSION}-cp27-none-linux_x86_64.whl
+ADD whl/tensorflow-*.whl /tmp/
```

Comment:
It would be nice if all the docker files just work. But we do not need this do we? We will use virtualenv for normal builds anyway. We will want to test this only when we do a release, right? We can script that later and differently. We can simply build and upload the pips before building the docker images.